.config/notice-file/README.md

```diff
@@ -0,0 +1,28 @@
+# Notice.txt File Configuration
+
+We are automatically generating Notice.txt by using first-level dependencies of the project. The related pipeline uses `config.yaml` stored in this folder.
+
+
+## Configuration
+
+Sample:
+
+```
+title: "Mattermost Playbooks"
+copyright: "©2015-present Mattermost, Inc.  All Rights Reserved.  See LICENSE for license information."
+description: "This document includes a list of open source components used in Mattermost Playbooks, including those that have been modified."
```

Comment:
Same here.

Reply:
See above, but basically this is a "sample configuration".